Time-Card/DRV/ptp_ocp.c

```diff
@@ -3923,6 +3926,132 @@ config_write(struct file *filp, struct kobject *kobj,
 }
 static BIN_ATTR_RW(config, OCP_CONFIG_SIZE);
 
+static ssize_t
+art_config_read(struct file *filp, struct kobject *kobj,
+	    struct bin_attribute *bin_attr, char *buf,
+	    loff_t off, size_t count)
+{
+	struct ptp_ocp *bp = dev_get_drvdata(kobj_to_dev(kobj));
+	struct nvmem_device *nvmem = ptp_ocp_nvmem_device_get(bp, NULL);
```

Comment:
Move the ppt_ocp_nvmem_device_get() call so it is just before the IS_ERR() statement.

Reply:
done it for both parameters